.travis.yml

```diff
@@ -38,4 +38,4 @@ script:
   - py.test
```

Comment:
We believe we'll need to change this line to only run pcsf tests in the pcsf environment.

This shows how to scope pytest: https://stackoverflow.com/questions/36456920/is-there-a-way-to-specify-which-pytest-tests-to-run-from-a-file

So we can modify this line to something like
```
pytest pcsf/tests/
```